news/atom.xml

```diff
@@ -14,8 +14,11 @@ layout:
     <name>The ev3dev team</name>
   </author>
 
+  {% assign index = 0 %}
```

Comment:
Yeah, I missed it too until I got into the code. I've amended my commit to rename that variable.